tests-integrations/fixtures/eslint-plugin-markdown-nest/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "test-for-eslint-plugin-jsonc-with-md-nest",
+  "private": true,
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "MIT",
+  "devDependencies": {
+    "eslint": "^8.19.0",
```

Comment:
I'm still not sure how are you testing for `ESLint@6`.

Reply:
Installing it in test.

https://github.com/ota-meshi/eslint-plugin-jsonc/pull/180/files#diff-5a830112b3744c98db597ca85f16ae74374c14ccdaabaee72a10f31443f34d25R38-R42

Reply:
Wow, I never imagined this functionality, could this be slow? (Personally I would choose to only test legacy verses on CI)

Reply:
This test is not run by regular test scripts (`npm test`).
Run `npm run test:integrations` to launch this test.
Also, the CI job is different. This test is slow, but I don't think it will have much impact on maintenance.